app/views/admin/pdfs/index.html.erb

```diff
@@ -1,12 +1,40 @@
-<div class="row">
-  <h1>This is where you can upload pdfs</h1>
-  <br>
-  <%= form_for :pdf, url: admin_pdfs_path, :html => { :multipart => true , class: "form-horizontal" } do |f| %>
-    <div class="form-group">
-      <%= f.label :pdf_file, "Upload PDF", class:"col-sm-2 control-group lead" %>
-      <%= f.file_field :pdf_file %>
-      <br>
-      <%= f.submit "Submit", class: "btn btn-primary col-sm-1" %>
+<h1>This is where you can upload pdfs</h1>
+
+<%= form_for :pdf, url: admin_pdfs_path, :html => { :multipart => true } do |f| %>
+    <%= f.label :pdf, "Upload PDF -->" %>
+    <%= f.file_field :pdf_file %> <br>
+    <%= f.submit "Submit" %>
+<% end %><br>
+
+<div class="container">
+  <h1 >Pdfs</h1>
+  <div class="row">
+    <div class="panel-heading">
     </div>
-  <% end %>
+      <table class="table">
+        <thead>
+          <tr class="filters">
+            <th><input type="text" class="form-control" placeholder="ID" disabled></th>
+            <th><input type="text" id="pdf_file_name" class="form-control" placeholder="Name"></th>
+            <th><input type="text" class="form-control" placeholder="Current Position" disabled></th>
+            <th><input type="text" class="form-control" placeholder="Last Updated" disabled></th>
+            <th><input type="submit" class="btn btn-default filter" id="filter-pdfs" value="Filter"></th>
+          </tr>
+        </thead>
+      <tbody class="all-pdfs">
+        <% @pdfs.each do |pdf| %>
+          <tr class="PDFs">
+            <td class="id"><%= pdf.id %></td>
+            <td class="name"><%= link_to pdf.pdf_file_file_name,
+                admin_pdf_path(pdf.id) %></td>
+            <td class="current-position"><%= pdf.slot %></td>
+            <td class="contact-title"><%= distance_of_time_in_words(pdf.updated_at, Time.now) %></td>
+            <td>
+              <%= link_to "Delete", admin_pdf_path(pdf.id), :method => "Delete", class: "btn btn-default" %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
```

Comment:
I hope that they will eventually send you to the pdf.  But as of now, nada.  JWAN!!!!